### PR TITLE
[011] Create basic sqlite using sqlalchemy

### PR DESCRIPTION
**Pull request by cassie-ying**

### CODE REVIEW

File: hypothesisfuzzer/app.py [hunk 1]
```diff
@@ -24,8 +30,19 @@ def get_commit_hash():
...
+
+"""
+SAMPLE CODE FOR A TABLE
+"""
+# class UserModel(db.Model):
+#     __tablename__ = 'users'
+
+#     uid = db.Column(db.Integer, primary_key=True)
+#     username = db.Column(db.String(80))
+#     password = db.Column(db.String(80))
```
Comment (jackel119) [inline hypothesisfuzzer/app.py]: remove comments pls

File: hypothesisfuzzer/app.py [hunk 2]
```diff
@@ -35,3 +52,10 @@ def __init__(self):
 
     def run(self, **kwargs):
         self.app.run(**kwargs)
+        db.create_all()
+
+
+"""
+FOR TESTING PURPOSE
+"""
+# Fuzzer().run(port=5000)
```
Comment (jackel119) [inline hypothesisfuzzer/app.py]: again, remove commented code

**Comment by jackel119** [comment]
Also need to merge develop down to this branch too

File: hypothesisfuzzer/app.py (hunk 1 above)
Comment (cassie-ying) [reply hypothesisfuzzer/app.py]: It is a sample use case of this database. I think it's good to keep it there.

**Pull request merged**